utils/article.py

```diff
@@ -21,3 +22,243 @@ def convertor(article):
         if isinstance(article_data, list):
```

Comment:
nice, using multiple inheritance really helped clean up and declutter the database.py file.
this